CLI tools/CurseForge to MultiMC-Git.py

```diff
@@ -9,28 +9,28 @@
 
 # Functions
 
-def remove_dir(path, description):
+def remove_dir(path: str, description: str) -> None:

```

Comment:
What does `-> None` mean here?

Reply:
It's return type. Just a good practice to specify types (params, return) even if it returns nothing (None, in this case)